config/application.rb

```diff
@@ -39,6 +39,38 @@ class Application < Rails::Application
 
     I18n.available_locales = ['en', 'sv']
 
+
+
+    ###
+    #
+    # Mail
+    #
+
+    config.action_mailer.delivery_method = :mailgun
+
+    #  need to set the mailgun_settings here because of  https://github.com/mailgun/mailgun-ruby/issues/86
+    config.action_mailer.mailgun_settings = {
+        api_key: ENV['MAILGUN_API_KEY'],
+        domain: ENV['MAILGUN_DOMAIN']
+    }
+
+
+    # Mail templates will need to use "_url" helpers rather than "_path" helpers
+    # since the template will not have the context of a request
+    # (as a controller does) and thus the full URL will be required to create
+    # links in the email.  This setting defines the host (domain) for the URL.
```

Comment:
Actually, no, your wiki is still absolutely correct.   :-)    `config/application.rb` now defines the default to be `localhost` and then the `production.rb` file overrides that to use the `ENV` setting.